charts/prefect-worker/values.yaml

```diff
@@ -55,6 +55,22 @@ worker:
     # -- additional sidecar containers
     extraContainers: []
 
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+  # -- optional dns policy for worker deployment
+  dnsPolicy: ""
+
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-dns-config
+  # -- optional dns config for worker deployment
```

Comment:
```suggestion
```

Ah, we can probably leave the top-level key out of the comments because it makes a bit of a weird entry in the readme.